admin/login.php

```diff
@@ -1,56 +1,53 @@
 <?php
-include 'common.php';
+require 'common.php';
 
 if ($user->hasLogin()) {
     $response->redirect($options->adminUrl);
 }
+
 $rememberName = htmlspecialchars(\Typecho\Cookie::get('__typecho_remember_name', ''));
 \Typecho\Cookie::delete('__typecho_remember_name');
 
-$bodyClass = 'body-100';
-
 include 'header.php';
 ?>
+
 <div class="typecho-login-wrap">
     <div class="typecho-login">
         <h1><a href="https://typecho.org" class="i-logo">Typecho</a></h1>
-        <form action="<?php $options->loginAction(); ?>" method="post" name="login" role="form">
+        <form action="<?= $options->loginAction(); ?>" method="post" name="login" role="form">
```

Comment:
Users can turn off short tags, so this modification is not necessarily appropriate.